index.js

```diff
@@ -38,6 +38,9 @@ gulp.task('build:dev', function (callback) {
 			'static:hb',
 			'static:hb:indexr'
 		],
+		[
+			'browserSupport'
```

Comment:
Why is this task separated from the others? I think this can be run together with static:hb tasks